reflekt-plugin/src/main/kotlin/io/reflekt/plugin/analysis/psi/annotation/Util.kt

```diff
@@ -7,6 +7,7 @@ import org.jetbrains.kotlin.psi.KtAnnotationEntry
 import org.jetbrains.kotlin.resolve.BindingContext
 import org.jetbrains.kotlin.resolve.lazy.ForceResolveUtil
 
+// FIXME problems with external libraries
```

Comment:
Which problem?

Reply:
Well, I faced the following problem: in the example with `librariesToIntrospect = listOf("org.junit.jupiter:junit-jupiter-api:5.7.0")` it failed to find descriptor for annotation entry `@API(status = EXPERIMENTAL, since = "5.5")` somewhere in the library and crashed with NPE.